cmd/tailscaled/tailscaled.go

```diff
@@ -115,6 +116,7 @@ func main() {
 	flag.StringVar(&args.socketpath, "socket", paths.DefaultTailscaledSocket(), "path of the service unix socket")
 	flag.StringVar(&args.birdSocketPath, "bird-socket", "", "path of the bird unix socket")
 	flag.BoolVar(&printVersion, "version", false, "print version information and exit")
+	flag.StringVar(&args.kubesecret, "kube-secret", "", "use kubernetes secrets as the state store")
```

Comment:
this description sounds like a bool argument.  What is the string value? Also, capitalize Kubernetes, no?

Reply:
But from https://github.com/tailscale/tailscale/pull/1919 I think the plan we were somewhat converging on was to use magic state path prefixes to mean different state backends.

In any case, we should have a plan for all three.

If we do separate flags for each, then we we need check that they're all mutually exclusive.

Reply:
Moved to `statepath` with `kube:` as a prefix